IFComp/lib/IFComp/Schema/Result/Entry.pm

```diff
@@ -1030,7 +1030,9 @@ sub update_content_directory {
 sub _create_parchment_page {
     my $self = shift;
 
-    my $title = $self->title;
+    my $title      = $self->title;
+    my $safe_title = $title;
+    $safe_title =~ s/'/\'/g;
```

Comment:
I think this needs to be:

```
$safe_title =~ s/'/\\'/g;
```

Where `\\` gets resolved into a literal backslash character.

An alternate, and maybe safer option, especially if recording_label isn't user-exposed data: just strip those quotes out entirely.

```
$safe_title = s/'//g;
```

Reply:
Yeah stripping them out entirely makes sense.